docs/validator/run/baklava-ledger.md

```diff
@@ -0,0 +1,635 @@
+
+```bash
+# On your local machine
+export CELO_VALIDATOR_GROUP_ADDRESS=<YOUR-VALIDATOR-GROUP-ADDRESS>
+export CELO_VALIDATOR_ADDRESS=<YOUR-VALIDATOR-ADDRESS>
+```
+
+### Start your Accounts node
+
+Next, we'll run a node on your local machine so that we can use these accounts to lock CELO and authorize the keys needed to run your validator.
+
+To run the node:
+
+```bash
+# On your local machine
+mkdir celo-accounts-node
+cd celo-accounts-node
+docker run --name celo-accounts -it --restart always --stop-timeout 300 -p 127.0.0.1:8545:8545 -v $PWD:/root/.celo $CELO_IMAGE --verbosity 3 --syncmode full --http --http.addr 0.0.0.0 --http.api eth,net,web3,debug,admin,personal --baklava --light.serve 0 --datadir /root/.celo
+```
+
+:::danger
+
+**Security**: The command line above includes the parameter `--http.addr 0.0.0.0` which makes the Celo Blockchain software listen for incoming RPC requests on all network adaptors. Exercise extreme caution in doing this when running outside Docker, as it means that any unlocked accounts and their funds may be accessed from other machines on the Internet. In the context of running a Docker container on your local machine, this together with the `docker -p 127.0.0.1:localport:containerport` flags allows you to make RPC calls from outside the container, i.e from your local host, but not from outside your machine. Read more about [Docker Networking](https://docs.docker.com/network/network-tutorial-standalone/#use-user-defined-bridge-networks) here.
+
+:::
+
+### Deploy a Validator
+
+To actually register as a validator, we'll need to generate a validating signer key. On your Validator machine (which should not be accessible from the public internet), follow very similar steps:
+
+```bash
+# from machine with ledger
+celocli account:list --useLedger --ledgerCustomAddresses "[2]" # vote signer address
+```
+
+#### Proof-of-Possession
+
+:::info
+
+Please complete this step if you are running a validator on Celo for the first time.
+
+:::
+
+In order to authorize our Validator signer, we need to create a proof that we have possession of the Validator signer private key. We do so by signing a message that consists of the Validator account address. To generate the proof-of-possession, run the following command:
+
+```bash
+# On the validator machine
+# Note that you have to export CELO_VALIDATOR_ADDRESS on this machine
+export CELO_VALIDATOR_ADDRESS=<CELO-VALIDATOR-ADDRESS>
+docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE account proof-of-possession $CELO_VALIDATOR_SIGNER_ADDRESS $CELO_VALIDATOR_ADDRESS
+```
+
+Save the signer address, public key, and proof-of-possession signature to your local machine:
+
+```bash
+# On your local machine
+export CELO_VALIDATOR_SIGNER_ADDRESS=<YOUR-VALIDATOR-SIGNER-ADDRESS>
+export CELO_VALIDATOR_SIGNER_SIGNATURE=<YOUR-VALIDATOR-SIGNER-SIGNATURE>
+export CELO_VALIDATOR_SIGNER_PUBLIC_KEY=<YOUR-VALIDATOR-SIGNER-PUBLIC-KEY>
+```
```

Comment:
I think some clarity here will go a long way. For example, which address should be set to  CELO_VALIDATOR_ADDRESS? Considering it was set in a previous step. 

Here is an example digest of the above command
```
Public address of the key:   0x74b4391F28bA889816A3a5b19c06bFfe15c02564
Path of the secret key file: /root/.celo/keystore/UTC--2023-06-07T19-51-45.356470709Z--74b4391f28ba889816a3a5b19c06bffe15c02564
```

Reply:
still trying to figure this out myself, as the next command yields an error 'Fatal: Could not find signer account ..'

Reply:
edit: just delete this line 

`export CELO_VALIDATOR_ADDRESS=<CELO-VALIDATOR-ADDRESS>`